packages/jaeger-ui/src/components/Monitor/ServicesView/index.tsx

```diff
@@ -123,8 +123,10 @@ const calcDisplayTimeUnit = (serviceLatencies: ServiceMetricsObject | ServiceMet
 };
 
 // export for tests
-export const yAxisTickFormat = (timeInMS: number, displayTimeUnit: string) =>
-  convertToTimeUnit(timeInMS * 1000, displayTimeUnit);
+export const yAxisTickFormat = (timeInMS: number, displayTimeUnit: string) => {
+  const formattedValue = convertToTimeUnit(timeInMS * 1000, displayTimeUnit);
+  return formattedValue.toString();
+};
```

Comment:
I think we can omit the use of `return formattedValue.toString();` and simply return `formattedValue`.
Is there any specific reason for returning a string value?

Reply:
> I think we can omit the use of `return formattedValue.toString();` and simply return `formattedValue`. Is there any specific reason for returning a string value?

Because in recharts the tickFormatter function expect sting | undefined. BaseAxisProps.tickFormatter?: ((value: any, index: number) => string) | undefined